addons/point_of_sale/models/pos_load_mixin.py

```diff
@@ -28,3 +28,6 @@ def _server_date_to_domain(self, domain):
         if last_server_date and domain is not False and limited_loading:
             domain = AND([domain, [('write_date', '>', last_server_date)]])
         return domain
+
+    def unrelevant_records(self):
+        return self.filtered(lambda product: not product.active).ids
```

Comment:
As it is now in pos_load_mixin, I suggest the lambda variable is now called record instead of product
```suggestion
        return self.filtered(lambda record: not record.active).ids
```